frontend/src/libs/components/backward-button/backward-button.tsx

```diff
@@ -0,0 +1,18 @@
+import { IconColor } from '#libs/enums/icon-color.enum.js';
+
+import { Icon } from '../components.js';
+import styles from './styles.module.scss';
+
+type Properties = {
+  onGoBack: () => void;
+};
+
+const BackwardButton: React.FC<Properties> = ({ onGoBack }) => {
```

Comment:
let's rename it to backButton as on mobile
```suggestion
const BackButton: React.FC<Properties> = ({ onGoBack }) => {
```

Reply:
done